nxdrive/gui/view.py

```diff
@@ -534,6 +611,8 @@ def data(self, index: QModelIndex, role: int = REMOTE_PATH) -> Any:
             return self.tr(label, args)
         elif role == self.PROGRESS:
             return f"[{row['uploaded']:,} / {row['planned_items']:,}]"
+        elif role == self.SHADOW:
+            return False
```

Comment:
Perhaps could you add a comment to say why this value is always set to `False` and why it is necessary to add this code?